core-services/prow/02_config/quay/quay-bridge-operator/_prowconfig.yaml

```diff
@@ -0,0 +1,22 @@
+tide:
+  queries:
+  - includedBranches:
+    - main
+    - master
+    - redhat-3.4
+    - redhat-3.5
+    - redhat-3.6
+    - redhat-3.7
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - keep-main-query-separate
```

Comment:
@petr-muller hm, it doesn't look like a label, but like a comment for authors like me, do you know what's motivation behind it? :)

Reply:
Hehe, I do. Feel free to remove it.

Back in the days when we had Tide config in a single file (not in repo-specific shards), we added auto-coalescing queries that had same merge criteria, to keep the gargantuan config smaller. But it also coalesced queries for main/master branches with relesae branch ones at occassions, because they change over time: sometimes they are identical, sometimes they are different. So I added this "must not be present to allow merge" fake to prevent the coalescer from merging my two queries :D